modules/tinymce/src/core/main/ts/init/Init.ts

```diff
@@ -156,11 +156,10 @@ const augmentEditorUiApi = (editor: Editor, api: Partial<EditorUiApi>) => {
   const uiApiFacade: EditorUiApi = {
     show: Optional.from(api.show).getOr(Fun.noop),
     hide: Optional.from(api.hide).getOr(Fun.noop),
-    disable: Optional.from(api.disable).getOr(Fun.noop),
-    isDisabled: Optional.from(api.isDisabled).getOr(Fun.never),
-    enable: () => {
+    isEnabled: Optional.from(api.isEnabled).getOr(Fun.always),
+    setEnabled: (state) => {
       if (!editor.mode.isReadOnly()) {
-        Optional.from(api.enable).map(Fun.call);
+        Optional.from(api.setEnabled).map((f) => f(state));
```

Comment:
```suggestion
        Optional.from(api.setEnabled).each((f) => f(state));
```
This wasn't something done here, but was incorrect originally as we aren't doing anything with the return value.